mathesar_ui/src/sections/table-view/header/HeaderCellDropdownTypeOptions.svelte

```diff
@@ -0,0 +1,225 @@
+<script context="module" lang="ts">
```

Comment:
While module context does make it easier to read/review components when we have static functions, it is encouraged to move static functions to stores or dedicated utils.

All the methods in this module context could essentially be moved to the databases store, or maybe even a dedicated store for database types. It would make our codebase more modular.

### Where we use module context:
We generally only prefer the module context, when there is a static variable or method for a component, which absolutely has to reside within the component.

You can see usages of it in components like components/.../Select.svelte in our repo. Here:
* We use module context to generate unique ids, essential for the component accessibility.
* This is an integer that is incremented per component initialization because we want it to remain the same during snapshot tests, and generating an uuid would make it impossible.
* This cannot be moved to a util, because the id static variable needs to be stored and incremented on the component context, but not on the instance, and certainly not on a global context.